### DISCUSSION TITLE
Softube Plugins on Audiogridder / Mac

### DISCUSSION THREAD

**Discussion by ThomBuck** [apohl79/audiogridder#526] [Help]
Don't Softube Plugins work with audiogridder? When I load one of them, audiogridder crashes...

**Comment by CaBuffalo** [comment]
Hi, Thomas, what’s your configuration?

**Reply by ThomBuck** [reply]
Hi CaBuffalo, its audiogridder 1.1.1 on a macbook pro 2012 with macOS Catalina as server and a macbook pro 2019 as client (same audiogridder client version).

**Comment by CaBuffalo** [comment]
I have Tape plugin. Let me test it when I get home later today.

On Wed, Apr 28, 2021 at 7:48 AM Thomas Buck ***@***.***>
wrote:

> Hi CaBuffalo, its audiogridder 1.1.1 on a macbook pro 2012 with macOS
> Catalina as server and a macbook pro 2019 as client (same audiogridder
> client version).
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/526#discussioncomment-670382>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6FN3Q6B6NGRMID4A2LTLAN5NANCNFSM43XFZHMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by ThomBuck** [reply]
cool. ja, tape plugin and harmonic esp. don't work.

**Comment by apohl79** [comment]
I can't confirm their full set of plugins, but drawmer and saturation knob work fine. Try different plugin formats. I don't recall but it could be that some of them don't work.

**Reply by ThomBuck** [reply]
i will try different formats as well, thanks for the hint.

**Comment by ThomBuck** [comment]
Hi CaBuffalo and apohl79 - The problem appears when using vst3 softube plugins. When using the AU versions no crashes occur. Only the ui doesn't reacts sometimes when klicking a button in the plugin - nothing happens. but by closing and opening the plugin the ui is clickable again. Thanks for your help. Audiogridder is great! I love it! Keep on doing such great work!

**Comment by CaBuffalo** [comment]
"Only the ui doesn't reacts sometimes when klicking a button in the plugin
- nothing happens." --- this happens to me as well. Waiting for a fix later.

Thanks for sharing this. Enjoy, cheers.

On Wed, Apr 28, 2021 at 2:31 PM Thomas Buck ***@***.***>
wrote:

> Hi CaBuffalo and apohl79 - The problem appears when using vst3 softube
> plugins. When using the AU versions no crashes occur. Only the ui doesn't
> reacts sometimes when klicking a button in the plugin - nothing happens.
> but by closing and opening the plugin the ui is clickable again. Thanks for
> your help. Audiogridder is great! I love it! Keep on doing such great work!
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/526#discussioncomment-672229>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HIUVBAUDHMF35T56LTLB5EPANCNFSM43XFZHMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by namooz** [comment]
I have done some testing with Softube and audiogridder, (just purchased another). I've had the same problem w/VST's. The plugins do load, but you must use the generic audio gridder platform before you load the Softube plugin. It is a skin issue with Softube I believe.

**Comment by CaBuffalo** [comment]
The experience I have now is that the plugin works well after a Windows
update. (I assume you are using a Windows PC as a slave server.)

I don't know what's going on. Yes, this only happens with Softube plugins.

On Wed, May 5, 2021 at 11:21 PM namooz ***@***.***> wrote:

> I have done some testing with Softube and audiogridder, (just purchased
> another). I've had the same problem. The plugins do load, but you must use
> the generic audio gridder platform before you load the Softube plugin. It
> is a skin issue with Softube I believe.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/526#discussioncomment-702342>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BYOJFWYAWXTI46ZBTTMIYQBANCNFSM43XFZHMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by namooz** [comment]
Thanks! I should have mentioned, I'm on Mac. Have a great day.
    On Thursday, May 6, 2021, 02:11:02 AM CDT, CaBuffalo ***@***.***> wrote:  
 
 

The experience I have now is that the plugin works well after a Windows
update. (I assume you are using a Windows PC as a slave server.)

I don't know what's going on. Yes, this only happens with Softube plugins.

On Wed, May 5, 2021 at 11:21 PM namooz ***@***.***> wrote:

> I have done some testing with Softube and audiogridder, (just purchased
> another). I've had the same problem. The plugins do load, but you must use
> the generic audio gridder platform before you load the Softube plugin. It
> is a skin issue with Softube I believe.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/526#discussioncomment-702342>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BYOJFWYAWXTI46ZBTTMIYQBANCNFSM43XFZHMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by charlesramirezhof** [reply]
For me works well with windows Server and Mac Logic as Pluging... But only Vst2 Versions works fine... :=) (I try tu use softube console One and also Works Well)

**Comment by CaBuffalo** [comment]
No, VST3 format won't pass the sound. I tried Tape and Curve bender plugins.

On Fri, May 14, 2021 at 3:45 PM charlesramirezhof ***@***.***>
wrote:

> For me works well with windows Server and Mac Logic as Pluging... But only
> Vst2 Versions works fine... :=) (I try tu use softube console One and also
> Works Well)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/526#discussioncomment-740557>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6D4CAO47JMTT536CSLTNWRW7ANCNFSM43XFZHMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
